README.md

```diff
@@ -184,6 +189,8 @@ default values are designed to give sensible results.
         from.
      3: Concise version of 2, omitting details of individual motif matches.
      4: Same than 1, but all info on one line.
```

Comment:
Is this output type replaced by the new one ("sort all clusters by score")?

Reply:
Yes, it is not supposed to be there. I pushed a new commit that fixes it.